_sections/30-projects.md

```diff
@@ -14,6 +14,7 @@ These projects pledge to drop Python 2 support in or before 2020.
 - [![](assets/jupyter.png)Jupyter notebook](https://jupyter.org)
 - [![](assets/numpylogoicon.svg)NumPy](http://numpy.org)
 - [![](assets/pandas.png)pandas](http://pandas.pydata.org/)
+- [![](assets/xarray.png)xarray](http://xarray.pydata.org/)
```

Comment:
We've generally tried to arrange projects by popularity, using Github stars as a rough proxy. So could we ask you to put this line just below sunpy, which has a similar number of stars to xarray.

I guess you were going next to pandas because of the similarity, which makes sense as well. But as I've already told a number of other people about this ordering, I don't want to play favourites. :-)

(Of course, not all projects are on Github, and some of them have probably got more popular since they were added, so this doesn't work very neatly...)

Reply:
Oops, I was looking at forks, not stars. Looking at stars, Xarray is just after Astropy.

Reply:
yes, I had set it next to pandas due to subject proximity but I get the ordering scheme now and have adjusted it accordingly.